Doc/whatsnew/3.13.rst

```diff
@@ -2005,231 +2006,510 @@ CPython Bytecode Changes
 C API Changes
 =============
 
+
 New Features
 ------------
 
-* You no longer have to define the ``PY_SSIZE_T_CLEAN`` macro before including
-  :file:`Python.h` when using ``#`` formats in
-  :ref:`format codes <arg-parsing-string-and-buffers>`.
-  APIs accepting the format codes always use ``Py_ssize_t`` for ``#`` formats.
-  (Contributed by Inada Naoki in :gh:`104922`.)
+* Add the :ref:`PyMonitoring C API <c-api-monitoring>`
+  for generating :pep:`669` monitoring events:
+
+  * :c:type:`PyMonitoringState`
```

Comment:
In case of nested lists, I think that it is better to use different marks, for example "-".
```rst
  - :c:type:`PyMonitoringState`
```

Reply:
I may be wrong here. Seems that there is no general preference to use different marks, and in the Sphinx documentation they use the same mark for nested bullet lists.

Reply:
Within this document we use ``*`` for second-level lists -- I just went with it for consistency.